test/partiql-tests-runner/src/main/kotlin/org/partiql/runner/ConformanceComparison.kt

```diff
@@ -11,91 +11,208 @@ fun main(args: Array<String>) {
+| :large_orange_diamond: Ignored | $firstIgnoreSize | $secondIgnoreSize | ${secondIgnoreSize - firstIgnoreSize} |
+| Total Tests | $firstTotalSize | $secondTotalSize | ${secondTotalSize - firstTotalSize} |
+                """.trimIndent()
+            )
+            this.appendLine(
+                """
 Number passing in both: ${passingInBoth.count()}
 
 Number failing in both: ${failingInBoth.count()}
 
-Number passing in Base ($origCommitId) but now fail: ${passingOrigFailingNew.count()}
+Number passing in Base (${first.commitId}) but now fail: ${passingFirstFailingSecond.count()}
+
+Number failing in Base (${first.commitId}) but now pass: ${failureFirstPassingSecond.count()}
+                """.trimIndent()
+            )
+            if (passingFirstFailingSecond.isNotEmpty()) {
+                // character count limitation with comments in GitHub
+                // also, not ideal to list out hundreds of test names
+                if (passingFirstFailingSecond.size < 10) {
+                    this.appendLine(":interrobang: CONFORMANCE REPORT REGRESSION DETECTED :interrobang:. The following test(s) were previously passing but now fail:\n<details><summary>Click here to see</summary>\n\n")
+
+                    passingFirstFailingSecond.forEach { testName ->
+                        this.appendLine("- $testName")
+                    }
+                    this.appendLine("</details>")
+                } else {
+                    this.appendLine(":interrobang: CONFORMANCE REPORT REGRESSION DETECTED :interrobang:")
+                }
+            }
+
+            if (failureFirstPassingSecond.isNotEmpty()) {
+                if (failureFirstPassingSecond.size < 10) {
+                    this.appendLine(
+                        "The following test(s) were previously failing but now pass. Before merging, confirm they are intended to pass: \n<details><summary>Click here to see</summary>\n\n"
+                    )
+                    failureFirstPassingSecond.forEach { testName ->
+                        this.appendLine("- ${testName}\n")
+                    }
+                    this.appendLine("</details>")
+                } else {
+                    this.appendLine("${failureFirstPassingSecond.size} test(s) were previously failing but now pass. Before merging, confirm they are intended to pass")
+                }
+            }
+        }
+}
 
-Number failing in Base ($origCommitId) but now pass: ${failureOrigPassingNew.count()}
-"""
-    )
+class CrossEngineReportAnalyzer(private val first: Report, private val second: Report) : ReportAnalyzer(first, second) {
+    override val reportTitle: String = "Conformance comparison report-Cross Engine"
+    override fun generateComparisonReport() =
+        buildString {
+            this.appendLine(
+                """### $reportTitle
+| | Base ${first.engine} | ${second.engine} | +/- |
```

Comment:
Could surround the first engine's name w/ parens

```suggestion
| | Base (${first.engine}) | ${second.engine} | +/- |
```